Makefile

```diff
@@ -46,6 +43,8 @@ bin_linux     = $(linux_dir)/$(bin_name)
 bin_darwin    = $(darwin_dir)/$(bin_name)
 bin_windows   = $(windows_dir)/$(bin_name)
 
+gox           = "github.com/mitchellh/gox"
```

Comment:
Why the change to how `gox` was being pulled in?

Reply:
Because I got this error when running `make`:

```
GO111MODULE=on go get -u github.com/mitchellh/gox
go get: installing executables with 'go get' in module mode is deprecated.
        To adjust and download dependencies of the current module, use 'go get -d'.
        To install using requirements of the current module, use 'go install'.
        To install ignoring the current module, use 'go install' with a version,
        like 'go install example.com/cmd@latest'.
        For more information, see https://golang.org/doc/go-get-install-deprecation
        or run 'go help get' or 'go help install'.
```

Reply:
I think you can borrow some inspiration from my other Makefiles (https://github.com/snowplow-devops/metrics-relay/blob/master/Makefile#L38-L40).  Have hit this issue and these ones work well up to Go 1.17

Reply:
The make file your linked to does not work for me.  If I run `make all` in metrics-relay I get this:

```
GO111MODULE=on go install github.com/mitchellh/gox@latest
mkdir -p build/compiled
GO111MODULE=on CGO_ENABLED=0 gox -osarch=linux/amd64 -output=build/output/linux/metrics-relay .
/bin/sh: line 1: gox: command not found
make: *** [Makefile:51: cli-linux] Error 127
```

I'm running go 1.17.6.

The make file I wrote here works, and I think it's future proof beyond 1.17.